t/admin/secrets.t

```diff
@@ -169,7 +169,69 @@ passed
 
 
 
-=== TEST 5: DELETE
+=== TEST 5: PATCH
+--- config
+    location /t {
+        content_by_lua_block {
+            local t = require("lib.test_admin").test
+            local etcd = require("apisix.core.etcd")
+            local res = assert(etcd.get('/secrets/vault/test1'))
+            local prev_create_time = res.body.node.value.create_time
+            assert(prev_create_time ~= nil, "create_time is nil")
+            local prev_update_time = res.body.node.value.update_time
+            assert(prev_update_time ~= nil, "update_time is nil")
+            ngx.sleep(1)
+
+            local code, body = t('/apisix/admin/secrets/vault/test1',
```

Comment:
Which field was modified? Didn't see how you verified the execution result of PATCH?

Reply:
TEST 4 changes the token to "unknown" by PATCH on path, then TEST 5 changes the token back to "apisix" by PATCH.